lib/private/Share20/Manager.php

```diff
@@ -1102,7 +1095,7 @@ public function moveShare(IShare $share, $recipientId) {
 	public function getSharesInFolder($userId, Folder $node, $reshares = false, $shallow = true) {
 		$providers = $this->factory->getAllProviders();
 		if (!$shallow) {
-			throw new \Exception('non-shallow getSharesInFolder is no longer supported');
+			throw new \Exception($this->l->t('non-shallow getSharesInFolder is no longer supported'));
```

Comment:
```suggestion
			throw new \Exception($this->l->t('Non-shallow getSharesInFolder is no longer supported'));
```
Or maybe `'"non-shallow" getSharesInFolder....`??

Reply:
To be honest, I think this message makes no sense to translate as this is an error message that does not help users. It is a developers only error, so maybe just not translate it?

```suggestion
			throw new \Exception('non-shallow getSharesInFolder is no longer supported');
```